docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+* Do users change URLs in a container tab? (sounds like it could be a flood unless we only record the first URL change?)
+
+### Follow-up Questions
+
+What are some follow-up questions we anticipate we will ask based on any of the
+above answers/data?
+
+* What is the average lifespan of a container tab? Is that longer or shorter than a regular tab? (if we don't have data on the latter, the former probably isn't worth gathering data on since we will have nothing to compare it to).
+
+## Data Collection
+
+### Server Side
+There is currently no server side component to Containers.
+
+### Client Side
+Containers will use Test Pilot Telemetry with no batching of data.  Details
+of when pings are sent are below, along with examples of the `payload` portion
+of a `testpilottest` telemetry ping for each scenario.
+
+* The user clicks on a container name to open a tab in that container
+
+```js
+  {
+    "userContextId": <userContextId>,
+    "clicked-container-tab-count": <number-of-tabs-in-the-container>,
+    "event": "container-tab-opened",
+    "eventSource": ["tab-bar"|"pop-up"]
+  }
+```
+
+* The user clicks "Edit Containers" in the pop-up
+
+```js
+  {
+    "event": "container-edit-containers"
+  }
+```
+
+* The user clicks OK after clicking on a container edit icon in the pop-up
+
+```js
+  {
+    "userContextId": <userContextId>,
+    "event": "container-edit-container"
+  }
+```
+
+* The user clicks OK after clicking on a container delete icon in the pop-up
+
+```js
+  {
+    "userContextId": <userContextId>,
+    "event": "container-delete-container"
+  }
+```
+
+* The user clicks OK after clicking to add a container in the pop-up
+
+```js
+  {
+    "userContextId": <userContextId>,
+    "event": "container-add-container"
+  }
+```
+
+* The user clicks the sort button/icon in the pop-up
+
+```js
+  {
+    "event": "container-sort-tabs",
+    "total-container-tabs-count": <number-of-all-container-tabs>
+  }
+```
+
+* The user clicks "Hide these container tabs" in the popup
+
+```js
+  {
+    "clicked-container-tab-count": <number-of-tabs-in-the-container>,
+    "event": "container-hide-tabs",
+    "hidden-containers-count": <number-of-containers-with-tabs-hidden>
+  }
+```
+
+* The user clicks "Show these container tabs" in the popup
+
+```js
+  {
+    "clicked-container-tab-count": <number-of-tabs-in-the-container>,
+    "event": "container-show-tabs"
+  }
+```
+
+* The user clicks "Move tabs to a new window" in the popup
+
+```js
+  {
+    "clicked-container-tab-count": <number-of-tabs-in-the-container>,
+    "event": "container-move-tabs-to-window"
+  }
+```
+
+### A Redshift schema for the payload:
+
+```lua
+local schema = {
+--   column name                    field type   length  attributes   field name
+    {"clickedContainerTabCount",    "INTEGER",   255,    nil,         "Fields[payload.clickedContainerTabCount]"},
+    {"container",                   "VARCHAR",   255,    nil,         "Fields[payload.container]"},
```

Comment:
This will be the userContextId, correct? What does this field measure? Is it a unique identifier for a user-context pair?

Looking through the example pings, it looks like most of the columns will be pretty sparse. Can we consistently include the userContextId so we can link container events?